packages/wallet/src/bitte-wallet.ts

```diff
@@ -200,11 +201,11 @@ export const BitteWallet: WalletBehaviourFactory<
     }
     const account = state.wallet.account();
 
-    return account.signAndSendTransaction({
+    return await account.signAndSendTransaction({
       receiverId: receiverId || contractId,
       actions: actions.map((action) => createAction(action)) as any,
       walletCallbackUrl: callback,
-    });
+    }) as any;
```

Comment:
Can we not do this?

Reply:
the changes you propose arent applicable.

<img width="516" alt="Screenshot 2024-07-29 at 11 22 46" src="https://github.com/user-attachments/assets/db3536a3-07a0-4b0f-8101-546ed6d87390">